### PR TITLE
Introduce new privacy questions

### PR DESCRIPTION
**Pull request by MKodde**
PART 2 of the Privacy Questions overhaul

In the second part, I added the new fields. Which are the `dpaType` and the multilingual `PrivacyStatementUrl` (nl+en) fields. They are added throughout the entire stack.

While at it, I found out the Privacy Questions where not included upon entity update actions. That was also addressed in this PR.

### CODE REVIEW

File: src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/DpaType.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+ *      "dpa_supplied_by_service",
+ *      "other"
+ *    ],
+ *    "default" : "service_has_own_dpa",
+ *    "info": "Determines what DPA this service has to offer"
+ *  }
+ */
+class DpaType
+{
+    public const DPA_TYPE_NOT_APPLICABLE = 'dpa_not_applicable';
```
Comment (pablothedude) [inline src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/DpaType.php:49]: Do all those constant have to be public?

File: src/Surfnet/ServiceProviderDashboard/Application/Command/PrivacyQuestions/PrivacyQuestionsCommand.php [hunk 2]
```diff
@@ -68,6 +69,18 @@ class PrivacyQuestionsCommand implements Command
      */
     private $otherInfo;
 
+    private DpaType $dpaType;
```
Comment (pablothedude) [inline src/Surfnet/ServiceProviderDashboard/Application/Command/PrivacyQuestions/PrivacyQuestionsCommand.php:72]: No assertion is needed here?

**Review by pablothedude — APPROVED** [review]

File: src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/DpaType.php (hunk 1 above)
Comment (MKodde) [reply src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/DpaType.php:49]: Good point, only the `default` was used outside of the VO scope. 

Updated

File: src/Surfnet/ServiceProviderDashboard/Application/Command/PrivacyQuestions/PrivacyQuestionsCommand.php (hunk 2 above)
Comment (MKodde) [reply src/Surfnet/ServiceProviderDashboard/Application/Command/PrivacyQuestions/PrivacyQuestionsCommand.php:72]: A required Assertion should be set here. Thanks for spotting that!

**Pull request merged**